src/model/value-objects/blueprint.ts

```diff
@@ -44,3 +45,12 @@ export interface BlueprintGenerateActions {
    */
   getMutateActionMethods?(action: Action): MutateActionMethods[]
 }
+
+export interface BlueprintValidateConfiguration {
+  /**
+   * Validates the configuration.
+   * Returns a list of StatusMessages where each StatusMessage is an error with the configuration.
```

Comment:
Is it alway error status message? It can't be good or warning?

Reply:
It can be whatever the `StatusCode` on `StatusMessage` is.
But StatusMessage that are GOOD will be ignored.

"error" here should be interpreted as something is wrong with the configuration.